modules/database-migrations/templates/migration_lambda.json.tpl

```diff
@@ -0,0 +1,38 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "rds-db:connect",
+      "Resource": [
+        "arn:aws:rds-db:eu-west-2:${account_id}:dbuser:${cluster_id}/migrations_user"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject",
+        "s3:ListBucket",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:s3:::tdr-database-migrations/*",
+        "arn:aws:s3:::tdr-database-migrations",
+        "${log_group_arn}"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DeleteNetworkInterface",
+        "ec2:DescribeInstances",
+        "ec2:CreateNetworkInterface",
+        "ec2:AttachNetworkInterface",
+        "ec2:DescribeNetworkInterfaces",
+        "autoscaling:CompleteLifecycleAction"
+      ],
+      "Resource": ["*"]
```

Comment:
I know this line has just been moved from elsewhere, but do we need to revisit this and make it more specific so the migration lambda can't attach any network interface?